zephyr/src/logging/spotflow_cbor_output_context.c

```diff
@@ -0,0 +1,24 @@
+﻿#include "spotflow_cbor_output_context.h"
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(cbor_output_context_spotflow, CONFIG_SPOTFLOW_PROCESSING_BACKEND_LOG_LEVEL);
+
+int spotflow_cbor_output_context_init(struct spotflow_cbor_output_context** _context)
+{
+	__ASSERT_NO_MSG(_context != NULL);
+
+	struct spotflow_cbor_output_context* context =
+	    k_malloc(sizeof(struct spotflow_cbor_output_context));
+	__ASSERT_NO_MSG(_context != NULL);
```

Comment:
```suggestion
	if (context == NULL) {
		return -ENOMEM;
	}
```

`_context` was already checked few lines above, so I assume it's a typo.